### PR TITLE
 Daemon cannot run

### PR DESCRIPTION
**Pull request by kangqian623**
the vlog file in badger dir is so big that cause an error in lotus daemon,detail see the log
```
2020/05/28 09:36:08 file size: 1412968288 greater than 4294967295
github.com/dgraph-io/badger/v2/y.AssertTruef
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/y/error.go:62
github.com/dgraph-io/badger/v2.(*logFile).open
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/value.go:910
github.com/dgraph-io/badger/v2.(*valueLog).open
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/value.go:1120
github.com/dgraph-io/badger/v2.Open
 /home/qq/go/pkg/mod/github.com/dgraph-io/badger/v2@v2.0.3/db.go:380
github.com/ipfs/go-ds-badger2.NewDatastore
 /home/qq/go/pkg/mod/github.com/ipfs/go-ds-badger2@v0.1.0/datastore.go:137
github.com/filecoin-project/lotus/node/repo.(*fsLockedRepo).Datastore.func1
 /home/qq/桌面/lotus/node/repo/fsrepo.go:268
sync.(*Once).doSlow
 /usr/lib/go-1.14/src/sync/once.go:66
sync.(*Once).Do
 /usr/lib/go-1.14/src/sync/once.go:57
github.com/filecoin-project/lotus/node/repo.(*fsLockedRepo).Datastore
 /home/qq/桌面/lotus/node/repo/fsrepo.go:264
```

fix it by clear the database vlog file

### CODE REVIEW

File: node/repo/fsrepo.go [hunk 1]
```diff
@@ -267,11 +270,27 @@ func (fsr *fsLockedRepo) stillValid() error {
 	return nil
 }
 
+func (fsr *fsLockedRepo) verifyDatastoreVlog(p string) error {
+	err := filepath.Walk(p, func(_ string, info os.FileInfo, err error) error {
+		if !info.IsDir() && path.Ext(filepath.Join(p, info.Name())) == "vlog" && info.Size() > MaxSize {
+			er := os.Remove(filepath.Join(p, info.Name()))
```
Comment (RobQuistNL) [inline node/repo/fsrepo.go:276]: Are you sure you want to automatically remove it? Wouldn't it be better to maybe gzip it and rename it?

Comment (RobQuistNL) [reply node/repo/fsrepo.go:276]: I mean I don't know whats in the file, and I've had similar issues but still 🤔

**Comment by liangping** [comment]
I got this error after rename 000021.vlog(70G)
````
 loading hamt node bafy2bzacebbpkblrzapb4lbsvtnhxlm3rwl3vsaxvub7mg7kjwh5tdakxt5ac failed: blockstore: block not found
```

**Comment by whyrusleeping** [comment]
Newer badger settings should obviate the need for this.

**Pull request closed without merging**